scripts/build.sh

```diff
@@ -387,17 +398,15 @@ function gen_manifests {
     log_info "=== Generating Kubernetes manifests ==="
     
     local res_dir_path
-    if [[ -n "${args["non-quarkus"]:-}" ]]; then
-        res_dir_path="${args["workflow-directory"]}"
-        log_info "Using non-Quarkus layout: resources in project root"
-    else
-        res_dir_path="${args["workflow-directory"]}/src/main/resources"
-        log_info "Using Quarkus layout: resources in src/main/resources"
-    fi
+    res_dir_path="${args["workflow-directory"]}"
+    
     
     # Validate resource directory exists
     validate_directory "$res_dir_path" "Workflow resources"
-    
+
+    log_info "Remove any target folder from: $res_dir_path"
+    find $res_dir_path -type d -name target -exec rm -rf {} +
```

Comment:
why not adding the target folder to .dockerignore instead?
i don't think that the script should modify existing resources in the working dir (other than the generation of the manifests if using the default)

Is the purpose to avoid having a .dockerignore for the non-quarkus layout?